scripts/presubmit.sh

```diff
@@ -138,7 +162,7 @@ main() {
       'have you installed github.com/alecthomas/gometalinter?' || exit 1
 
     echo 'running gometalinter'
-    gometalinter --config=gometalinter.json ./...
+    retry 5 gometalinter --config=gometalinter.json --deadline=2m ./...
```

Comment:
Have you seen the bit in the metalinter FAQ about exit status?

https://github.com/alecthomas/gometalinter/blob/master/README.md#exit-status

Looks like you might be able to differentiate between a lint fail and a tool fail, so potentially you can early out when there really is a lint problem rather than retrying something which is my going to get fixed.

Reply:
Ah, neat – I've changed `retry` to only do so for rc=2.